How to get more people starting with programming (#11)
chhoumann/p4: Report/sections/1_analysis/Learning.tex

Research from Georgia Institute of Technology has shown a positive correlation between playing video games and the interest in computer science. The research included a survey which 1116 students participated in; the results shows that 43\% of student claim that playing games have increased their interest in computing by a lot. 
Even though the results from the study show a low interest in becoming a computer scientist (6.9\%), nearly 10\% (92 out of 1116 students) reported that playing video games sparked a “desire to make games”. 
This means that a fair number of young students has the desire to make games without necessarily being interesting in learning or studying computer science. Therefore, a simple and easy programming language that enables the user to quickly create and explore the possibilities in game development could be in demand.
As discussed in the previous section interest plays an important role in effective learning. Therefore learning a simple code language could be a good stepping stone to help understand a more complex languages.

\chapter{How to get more people starting with programming}\label{chap:analysis}

> This section will focus on how to get more people starting to program and what is the best ways to teach it. \newline

Reviewer: ```suggestion
This section will focus on how to get more people  to start programming and what the best way to teach it is. \newline
```

> Learn how to code is not easy, taking the first steps into this huge universe might seem like a daunting, if not intimidating task. All software is written in some kind of coding language and every coding language is unique, designed with a certain operating system, platform, coding style and intended use in mind. 

Reviewer: ```suggestion
Learning how to code is not easy, taking the first steps into this huge universe might seem like a daunting, if not intimidating task. All software is written in some kind of coding language and every coding language is unique, designed with a certain operating system, platform, coding style and intended use in mind. 
```

> That is why many have a hard time starting to program because the coding world can be complicated and confusing to get into. It takes years of pratice to learn everything necessary to get beyond just a basic level where you can write short programs that work. It takes hours of practice.

Reviewer: Hmm... Ændre det med at det "takes years of practice just to get to a basic level where you can write a short program".  Det tænker jeg ikke det gør. 

Måske skift det ud med den sidste sætning der "it takes hours of practice", da den alligevel virker lidt indskudt for sig selv. 

Evt. komme ind på at det både kan være abstrakt at lære til at starte med som ny, men også at learning to code har en meget stor læringskurve, som kan gøre det svært for begyndere at vide hvor de skal starte. Så derfor kan det være godt at de har noget konkret de gerne vil programmere, noget de f.eks. finder interessant såsom et spil, kunne være et godt sted at starte.

> There are plenty of free resources beginners can use to give themselves all the help they need. It can help them teach new techniques, and makes the learning process fun and exciting. 

Reviewer: ```suggestion
There are plenty of free resources beginners can use to give themselves all the help they need. It can help them teach new techniques, and make the learning process fun and exciting. 
```

> One of the most important ways before starting programming is to find a reason to learn a programmming language. The answer can help determine which programming language should be used whetever it is to make a new indie game or making the newest application.

Reviewer: ```suggestion
One of the most important ways before starting to program, is to find a reason to learn a programmming language. The answer can help determine which programming language should be used whetever it is to make a new indie game or making the newest application.
```

Reviewer: Evt. sæt dette afsnit sammen med det afsnit jeg foreslog tidligere, så de ligger i forlængelse af hinanden.  At det er en god ide at have noget konkret folk gerne vil lære at programmere...

> Another way to get the best learning expericence is to get a real book that you can follow along from beginning till the end. This will give more comprehensive introduction to coding than jumping around from topic to topic on a website. Another idea to start programming is to watch educational videos about nearly any coding related topic. 

Reviewer: Evt. beskriv lidt generelt, så det måske hænger sammen med noget af det andet, at det er en god ide at nye programmører har et 'klart/konkret' forløb de kan følge, netop for at gøre det mere overskueligt, da kodning som sagt er meget bredt og har en stor læringskurve.
Såsom f.eks. bøger som du nævner, eller kurser på hjemmesider (coding camp) eller en tutorial på youtube om et eller andet projekt.
